examples/wip/viewport-animation/3d-heatmap/README.md

```diff
@@ -0,0 +1,36 @@
+This is a minimal standalone version of the 3DHeatmap example
```

Comment:
I'd prefer a flat directory structure for the examples in `wip`, it will make it easier to see what is being built for the new release at a glance. E.g.:
```
examples/wip/viewport-animation-3d-heatmap
examples/wip/viewport-animation-geojson
```